azure-iothub-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/iothub/IoTHubProperties.java

```diff
@@ -0,0 +1,36 @@
+package com.microsoft.azure.autoconfigure.iothub;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("azure.iothub")
+public class IoTHubProperties {
+    @NotNull
```

Comment:
I'm never relying on annotations. Duplicate checks don't hurt and give much better log messages